cola/ace_generator.py

```diff
@@ -0,0 +1,90 @@
+from .ce_module.ce_models import DiCE
+from .model import Model
+from .data import Data
+from .policy_module import policy
+import numpy as np
+import pandas as pd
+
+class COLA:
+    def __init__(self, ml_model=None, data=None, counterfactual_algorithm=None, policy_name="CF_OTMatch", Avalues_method="max", limited_actions=None):
+        """
+        Initialize the ACE_generator class
+
+        Parameters:
+        ml_model: Machine learning model used for predictions
+        data: DataFrame containing the data
+        counterfactual_algorithm: Algorithm to generate counterfactual data
+        limited_actions: List of limited actions to consider
+        """
+        self.ml_model = ml_model
+        self.data = data
+        self.counterfactual_algorithm = counterfactual_algorithm
+        self.limited_actions = limited_actions
+        self.Avalues_method = Avalues_method
+        self.policy_name = policy_name
+        # self.ce_algorithm =None
+        # self.results = self.generate_counterfactual()
+
+    def generate_ace_counterfactual(self):
```

Comment:
generate_action_constrained_counterfactual sounds better.

It is difficult to get the meaning of "ace"